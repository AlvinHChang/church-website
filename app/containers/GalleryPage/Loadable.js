/**
 * Asynchronously loads the component for GalleryPage
 */

import loadable from 'utils/loadable';

export default loadable(() => import('./index'));
