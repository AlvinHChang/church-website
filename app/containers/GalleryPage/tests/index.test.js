import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import GalleryPage from '../index';

describe('<GalleryPage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <GalleryPage />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
