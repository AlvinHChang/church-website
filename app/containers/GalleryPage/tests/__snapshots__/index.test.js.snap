// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GalleryPage /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the GalleryPage container!
  </span>
</h1>
`;
